relay-cardinality/src/statsd.rs

```diff
@@ -8,7 +8,7 @@ pub enum CardinalityLimiterCounters {
     ///  - `scope`: The scope of check operation.
     #[cfg(feature = "redis")]
     Rejected,
-    /// Incremented for every loaded Redis set.
+    /// Incremented for every redis cardinality check.
```

Comment:
Maybe we should rename the entire metric? We dont have dashboards yet, but `Read` also somewhat fits, that is how many times we went to redis to read cardiniality data

Reply:
Is the number of times that we call the Lua script? Then maybe `RedisCheck` makes sense?

We could also get rid of `RedisHashCheck`, in favor of a histogram metric that tracks the number of hashes per check.

Reply:
I believe we can get this number from a histogram as well, right?

If we add the histrogram for the amount of hashes sent to redis, we can also get the count from the histogram. -> I'll switch to the histrogram